python/dashboard/feed.py

```diff
@@ -20,9 +20,11 @@
 import pandas as pd
 import matplotlib.pyplot as plt
 
-sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", "..", "lib"))
 from splunklib import client
 from splunklib import results
+
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
```

Comment:
I don't think this is needed?

Reply:
Yes. thanks for highlighting, we missed this. We have removed the extra line.